KWStyle.xml

```diff
@@ -0,0 +1,131 @@
+<Project>LMMS</Project>
```

Comment:
Please move this to `.travis`.  Also, I recommend making the filename lowercase, which is what we do for all non-class files.